### ISSUE TITLE
Accessibility: WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2

### ISSUE THREAD

**Issue by funkysi1701** [funkysi1701/funkysi1701.github.io#2824]
## Accessibility Issue Found by Pa11y

**WCAG Code**: `WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2`
**Total Occurrences**: 4
**URL**: https://blog-dev.funkysi1701.com
**Build**: 10.1.1.2333

---

### Example 1

**Message**: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
**Selector**: `#offcanvasSettings > div:nth-child(2) > section:nth-child(1) > form`

**Context:**
```html
<form class="row"><div class="col-auto"><label><i...</form>
```

### Example 2

**Message**: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
**Selector**: `#offcanvasSettings > div:nth-child(2) > section:nth-child(2) > form`

**Context:**
```html
<form class="row"><div class="col-auto"><label><i...</form>
```

### Example 3

**Message**: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.
**Selector**: `#navbarSupportedContent > form`

**Context:**
```html
<form class="search-bar my-1" action="https://blog-dev.funkysi1701.com:443/search" style="display:none"><div class="input-group input-g...</form>
```

---
_Detected by [Pa11y](https://pa11y.org/) accessibility checker in Azure Pipeline_
